docker-hub/release-notes.md

```diff
@@ -9,6 +9,14 @@ toc_max: 2
 Here you can learn about the latest changes, new features, bug fixes, and
 known issues for each Docker Hub release.
 
+# 2021-05-05
+
+### Enhancement
+
+The Advanced Image Management dashboard enables a new level of access to your content, allowing fine-grained pruning of old data and exploration of old versions of pushed images. For more information, see  [Advanced Management Dashboard](/image-management.md).
```

Comment:
```suggestion
When managing the content of your repositories, you can now filter the results based on the currentness of the tags and more easily identify your untagged images.
For more information, see  [Advanced Management Dashboard](/image-management.md).
```
I'd like us to be more specific about describing the enhancement within Advanced Image Management (dashboard and API), because it is quite discrete and we want to help our users notice it.
@jerae-duffin Don't hesitate to improve the readability of my suggestion :-) . TIL 'currentness' _is_ an actual word...